hyper/pkg/proto/hyper.proto

```diff
@@ -52,3 +56,28 @@ message DesktopMigrateRequest {
 }
 
 message DesktopMigrateResponse {}
+
+message DesktopResumeRequest {
+    string id = 1;
+}
+
+message DesktopResumeResponse {}
+
+message DesktopSuspendRequest {
+    string id = 1;
+}
+
+message DesktopSuspendResponse {}
+
+message DesktopRestoreRequest {
+    string path = 1;
+}
+
+message DesktopRestoreResponse {}
+
+message DesktopSaveRequest {
+    string id = 1;
+    string path = 2;
+}
+
+message DesktopSaveResponse {}
```

Comment:
This should follow the same order as in the service definition. Also, there should be a newline at the end of the file

Reply:
ok